docs/product_release.md

```diff
@@ -0,0 +1,18 @@
+# Product Release
+
+## Artifacts
+
+- Container images
+- Helm Chart
```

Comment:
shall we call these "Product Helm Chart" => to differentiate from "Umbrella HC" in TX-Release?

Reply:
Product Helm Chart with Dependencies

Reply:
Would "Helm Chart (referred to as product Chart)" be fine?. 
Both, the Tractus-X Chart, as well as single product Charts do have dependencies. 
Some do only include a Database dependency, but some also provide frontend and backend as separate Charts, that are then referenced in a product umbrella Chart